dev/baked/init.sh

```
@@ -0,0 +1,21 @@
+#!bin/bash
```

Shellcheck (0.10.0)

[error] Ensure the shebang uses an absolute path to the interpreter.

(SC2239)